.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+# Main workflow for testing the Lingua Franca compiler.
+name: CI
+
+on:
+  # Trigger this workflow on push events, but only on master.
+  push:
+    branches:
+      - master
+  # Trigger this workflow also on pull_request events, but ignore the 'nightly' branch.
+  pull_request:
+    branches-ignore:
+      - 'nightly'
+
+env:
+  # 2020.11
+  vcpkgGitRef: 0bf3923f9fab4001c00f0f429682a0853b5749e0
+
+jobs:
+  # Run the unit tests.
+  unit-tests:
+    uses: lf-lang/lingua-franca/.github/workflows/unit-tests.yml@modular-ci
```

Comment:
I assume we will have to delete the `@modular-ci` part after this is merged(?)

Reply:
We could just change it to `@master` if we want to remove the `modular-ci` branch.